lib/api/v3/queries/query_representer.rb

```diff
@@ -185,6 +185,8 @@ def initialize(model,
 
         property :timeline_visible
 
+        property :show_hierarchies
```

Comment:
I don't find `hierarchies` appropriate. I could settle with `hierarchyMode`, since we'll likely have other modes in the future.